tests/runtime/uprobe

```diff
@@ -52,6 +52,18 @@ EXPECT Attaching 1 probe...
 TIMEOUT 5
 BEFORE ./testprogs/mountns_wrapper uprobe_test
 
+NAME uprobes - attach to probe by pid with only wildcard
+RUN {{BPFTRACE}} -e 'uprobe:*:function1 { printf("here\n" ); exit(); }' -p {{BEFORE_PID}}
+EXPECT Attaching 1 probe...
+TIMEOUT 5
+BEFORE ./testprogs/uprobe_test
+
+NAME uprobes - attach to multiple probes by pid with only wildcard
+RUN {{BPFTRACE}} -e 'uprobe:*:function* { printf("here\n" ); exit(); }' -p {{BEFORE_PID}}
+EXPECT Attaching 4 probes...
+TIMEOUT 5
+BEFORE ./testprogs/uprobe_test
+
```

Comment:
Runtime test for wildcard filepath without `-p`?

Reply:
> Runtime test for wildcard filepath without `-p`?

This can vary greatly in terms of how long it takes to go over everything on the host (depending on what's running) so I was a little afraid of adding it - guess I'm having a similar problem for the usdt test.